beacon-chain/db/validator.go

```diff
@@ -72,9 +97,29 @@ func (db *BeaconDB) DeleteValidatorIndex(pubKey []byte) error {
 	})
 }
 
+// HasValidators checks if a validator index map exists out of a list of public keys.
```

Comment:
Please update this comment. This method returns true if any of the pubkeys exist in the database.

Reply:
this method returns false if any one of the slice element is missing

Reply:
// HasValidators returns true if all validator in a list of public keys are in the validator bucket.